lib/Transforms/Scalar/PromoteVerifierCalls.cc

```diff
@@ -159,7 +168,7 @@ bool PromoteVerifierCalls::runOnFunction(Function &F) {
 
       IRBuilder<> Builder(F.getContext());
       Builder.SetInsertPoint(&I);
-      CallInst *ci = Builder.CreateCall(m_failureFn);
+      CallInst *ci = Builder.CreateCall(m_errorFn);
```

Comment:
why was this changed from `m_failureFn` to `m_errorFn`?

Reply:
Sorry i was trying auto format on save in emacs... it messed up things... and i missed reverting this.